src/message/MessageContainer.js

```diff
@@ -37,7 +37,7 @@ export default class MessageContainer extends React.PureComponent {
         doNarrow={doNarrow}
         onPress={this.handlePress}
       >
-        {renderHtmlChildren({ childrenNodes: dom, auth })}
+        {renderHtmlChildren({ childrenNodes: dom, auth, doNarrow })}
```

Comment:
I would prefer to keep the `html` components rather dumb and bubble the behavior to the container.
So, they will not know how to narrow or what to do even on press.
The difference with 'doNarrow' will be that we handle the onPress.
It will still function almost the same, but is conceptually different.